src/lightning/pytorch/trainer/trainer.py

```diff
@@ -844,6 +844,13 @@ def _run(
             self.fit_loop.min_epochs = min_epochs
             self.fit_loop.max_epochs = max_epochs
 
+        if self.barebones:
+            # no progress bar in barebones can make it look like the Trainer hung
+            rank_zero_info(
+                "`Trainer(barebones=True)` started running. The progress bar is disabled so you might want to"
+                " manually print the progress in your hooks."
```

Comment:
```suggestion
                " manually print the progress in your model."
```

otherwise people might not know which hooks you mean